.github/workflows/pipeline.yml

```
@@ -0,0 +1,49 @@
+  push:
+    branches:
+      - pipeline-larivasd
+
+jobs:
+  deploy:
+    name: Deploy to EC2
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout the files
+      uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)